app/assets/stylesheets/application.css.scss

```diff
@@ -93,6 +93,12 @@ a, a:visited, a:link {
     text-align: center;
 }
 
+.flash_remove {
+  color: green;
+  text-align: center;
+  text-shadow: white 1px 1px 1px 1px;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.